User.java

```diff
@@ -0,0 +1,74 @@
+
+public class User {
+	/*For the User class the fields needed are:
+	 * Name, age, city, state, dissabilities, and
+	 * number of families.
+     * 
+     * I added simple setters and getters and some boolean methods 
+     * that decide whether or not the user is a minor
+	 * */
+	private String name;
+	private int age;
+	private String city;
+	private String state;
+	private String disabilities;
+	private int numOfFamMembers;
+	public User(String name, int age, String city, String state, String disabilities, int numOfFamMembers) {
+		this.name = name;
+		this.age = age;
+		this.city = city;
+		this.state = state;
+		this.disabilities = disabilities;
+		this.numOfFamMembers = numOfFamMembers;
+	}
+	public User() {
+		this.name = "Left Blank";
+		this.age = 18;
+		this.city = "Left Blank";
+		this.state = "Left Blank";
+		this.disabilities = "Left Blank";
+		this.numOfFamMembers = 0;
+    }
+    public void setName(String str){
+        name = str;
+    }
+    public String getName(){
+        return name;
+    }
+	public void setAge(int num) {
+		age = num;
+    }
+    public int getAge(){
+        return age;
+    }
+    public void setCity(String str){
+        city = str;
+    }
+    public String getCity(){
+        return city;
+    }
+    public void setState(String str){
+        state = str;
+    }
+    public String getState(){
+        return state;
+    }
+    public void setDisabilities(String str){
+        disabilities = str;
+    }
+    public String getDisabilities(){
+        return disabilities;
+    }
+    public void setNumFamilyMembers(int num){
+        numOfFamMembers = num;
+    }
+    public int getNumFam(){
+        return numOfFamMembers;
+    }
+    public boolean isAdult(){
```

Comment:
I like that you added methods for checking isAdult and haveFamMembers. Even thought this is just a simple boolean comparison, clients that use this class shouldn't have to worry about what age is considered an adult, or how family members are represented internally. Good idea!

Extra credit: Consider making a private static final field for ADULT_AGE and using that here. That way you can also refer to that constant value if you need to reason about the adult age elsewhere, and if you decide to change what you consider adult age (I could see an argument for 21), it's easy to go change it there and have it reflected everywhere that its used.